scripts/createdownloadurls.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
```

Comment:
Add a header comment here to explain what this script does.

Reply:
@conorsch I used `black` in the latest push.